docs/triggertemplates.md

```diff
@@ -48,11 +46,13 @@ spec:
         resources:
         - name: git-source
           resourceRef:
-            name: git-source
+            name: git-source-$(uid)
 ```
 
 Similar to [Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md),`TriggerTemplates` do not do any actual work, but instead act as the blueprint for what resources should be created.
 Any parameters defined in the [`TriggerBinding`](triggerbindings.md) are passed into to the `TriggerTemplate` before resource creation.
+The `$(uid)` variable can be used through resource templates, which appends a postfix like the `generateName` metadata field, but allows the template be well defined.
```

Comment:
> ...which appends a postfix like the `generateName` metadata field, but allows the template be well defined.

nit: This is unintentionally slightly inaccurate, because the `$(uid)` variable does not `append a postfix`. Instead, the `$(uid)` is being substituted with a unique string. This can be used to append a postfix, but it can also be used for other purposes.